### PR TITLE
Added \db command (issue #489).

### PR DESCRIPTION
**Pull request by j-bennet**
This adds `\db` command to address the following issue in pgcli:

https://github.com/dbcli/pgcli/issues/489

### CODE REVIEW

**Comment by j-bennet** [comment]
@amjith or @stuartquin or @darikg, would one of you have the time to review?

**Comment by amjith** [comment]
I'll have time today after work. Sorry for the delay. Having a hectic week (can't believe it's only Wed).

File: .travis.yml [hunk 1]
```diff
@@ -5,7 +5,8 @@ python:
   - "3.3"
   - "3.4"
 
-install: 
+install:
+  - psql --version
```
Comment (amjith) [inline .travis.yml]: What's the purpose for this?

**Comment by amjith** [comment]
The change looks fine and works great. Thank you! 🐯

**Pull request merged**